### ISSUE TITLE
Show validation accuracy on right-side mission panel

### ISSUE THREAD

**Issue by jonfroehlich** [ProjectSidewalk/SidewalkWebpage#1914]
Related to https://github.com/ProjectSidewalk/SidewalkWebpage/issues/1680:

Currently, there is no easy way for a user to know how well they are doing. What if we begin to show a user's validation accuracy *always* (and prominently) on the right-side mission panel.

![image](https://user-images.githubusercontent.com/1621749/66003545-43133580-e45b-11e9-94a6-f4e9eb3237c7.png)

**Comment by misaugstad** [comment]
Some thoughts:
1. This is a fun idea that is (hopefully) motivating to people. Ideally it would make them want to look more carefully :sweat_smile: 
1. I think if we are going to include a user's accuracy, we should also include the mean or median accuracy across all users for reference
1. If we do this, we should definitely have a tooltip on the accuracy number that explains that this number comes from validations from other Project Sidewalk users.
1. idk if I like giving them this accuracy number without a way for them to see the mistakes that they've made. I guess that if they see their number and it motivates them to improve their auditing, we could have a link to the labeling guide for now until we have something set up that let's them see what they've done wrong.
1. I worry a tiny bit about some users (turkers, in particular) recognizing that if they place labels on less obvious issues, other users are more likely to disagree which will bring down their accuracy rating. Like if you really think this through, placing fewer labels will result in a higher accuracy number, but we really want to be encouraging users to place _more_ labels, because it is much easier to remove false positives than false negatives. I'd like to think that _most_ users won't think through it that far though...

    One tiny way to address this is to just imply, through the wording of the tooltip, that looking around carefully for problems will improve their accuracy. Like it isn't really truthful, but most users will probably take our explanation at face value. We could say something like "Your accuracy is determined from other Project Sidewalk users through the validation interface. You can improve your accuracy by slowing down, checking both sides of the street carefully for problems, and reading through our labeling guide" (with a link to the labeling guide there).
1. We would need to require a minimum number of validations of their labels before we start showing a number. Maybe 20? It might be fun to analyze the data to figure out at what point a user's accuracy number starts to become stable.
1. Just a note to make sure to exclude validations from users manually marked as low quality (#2128)

**Comment by jonfroehlich** [comment]
Also described in this Issue with mocks: https://github.com/ProjectSidewalk/SidewalkWebpage/issues/2260#issuecomment-1074391381

**Comment by jonfroehlich** [comment]
Once Mikey finishes adding recent "mistakes" to the user dashboard (https://github.com/ProjectSidewalk/SidewalkWebpage/issues/2288), I think adding in accuracy to the side panel would be super helpful. Perhaps if they click on the accuracy, they get taken to their dashboard (which builds awareness of that feature and allows them to learn about why their accuracy may be low--so helps them learn!)

**Comment by jonfroehlich** [comment]
@misaugstad and I just discussed this and mocked this up together (but using icons from the dashboard for consistency)

![image](https://user-images.githubusercontent.com/1621749/187761220-3245e162-0e02-4226-b7b3-0b4a4353e591.png)

**Comment by misaugstad** [comment]
@jonfroehlich how does this look? I realized that it's kinda confusing to have stats that are for all of your work in the section called "current neighborhood", which makes it sounds like they are stats specific to this neighborhood!

![Screenshot from 2022-10-06 16-12-21](https://user-images.githubusercontent.com/6518824/194434898-56f92a69-7eb8-488e-8259-8c8562a57938.png)

Though it feels like overkill to have your neighborhood stats AND overall stats. Maybe remove the "Current Neighborhood" section altogether, and replace "explore 500 ft of this neighborhood" with "explore 500 ft of North Linden, Columbus"? This would make it a lot less busy!

**Comment by jonfroehlich** [comment]
Awesome. I think fine to have Overall Stats and Current Neighborhood stats.

For Current Neighborhood, make sure to use the same icons.

So, 
![image](https://user-images.githubusercontent.com/1621749/194441285-aef485b2-d3dd-49ac-83df-1751f456ac7c.png)
for distance

and 
![image](https://user-images.githubusercontent.com/1621749/194441306-a31a5cdd-be07-4bbd-b22e-6b8c77ae1d6d.png) for labels

**Comment by misaugstad** [comment]
Here's how we're looking now! 
![Screenshot from 2022-10-10 15-15-04](https://user-images.githubusercontent.com/6518824/194960370-11bcd1e3-447b-4c56-a853-4e2cf01c5eb4.png)

My question is... Should we add some of these stats to Validate? It won't be difficult, but we would just need to show stats that aren't confusing to people.
* Distance is not confusing
* A label count could be confusing. Right now we are showing the count of validated labels, and we could keep it like that. Or we could have both "labels" and "labels validated" counts
* Accuracy is the one that feels the most confusing. How does someone know if we are showing their own accuracy on their labels vs their accuracy _when validating_? Perhaps we could clarify in a tooltip.

![Screenshot from 2022-10-10 15-19-52](https://user-images.githubusercontent.com/6518824/194960832-d3eba72f-1858-4e92-bd27-43669eecd983.png)

**Comment by jonfroehlich** [comment]
Great work @misaugstad. 

- Could you make the icons a bit smaller (try shrinking by ~25% each) and the text slightly larger.
- I think we discussed hyperlinking "X.X% accuracy" so that it would take users to their dashboard.
- Just thinking: What do anonymous users see for their stats?
- While I think having more stats on the validation sidebar would be useful, I think we should put this into another Issue and PR (because it's not clear to me which stats to include and I agree that accuracy is overloaded here)

**Comment by misaugstad** [comment]
>Could you make the icons a bit smaller (try shrinking by ~25% each) and the text slightly larger.

Yep, here's how it's looking! I was a bit worried about the icons looking kinda pixelated, but they still seem decent at that size.

![Screenshot from 2022-10-10 15-52-20](https://user-images.githubusercontent.com/6518824/194963894-0502d72e-7b36-44c4-85f3-b306256f709b.png)

>I think we discussed hyperlinking "X.X% accuracy" so that it would take users to their dashboard.

Yep! I've also been considering a tooltip that says something like "Accuracy is based on validations from other users. If your accuracy is low, check out some recent mistakes on your User Dashboard to help you improve!" with "your User Dashboard" hyperlinked. Thoughts on that? 

>Just thinking: What do anonymous users see for their stats?

As of right now, they see whatever we have saved for them, so it's whatever work they did in this session. Could always have a tooltip inviting them to sign up since their accuracy is N/A :)

>While I think having more stats on the validation sidebar would be useful, I think we should put this into another Issue and PR (because it's not clear to me which stats to include and I agree that accuracy is overloaded here)

Sounds good!

**Comment by jonfroehlich** [comment]
Looks better imo. Note that the text next to the icons is not perfectly vertically centered.

![image](https://user-images.githubusercontent.com/1621749/194964656-35331276-63ac-4ede-a11d-7ed64a0056e5.png)

Re: tooltip. Sounds good

**Comment by jonfroehlich** [comment]
Oh, and the icons are all SVGs, so we can generate pixel-perfect versions of any size. Just lmk.
